zanata-maven-plugin/demo-props/pom.xml

```diff
@@ -14,7 +14,7 @@
       <plugin>
         <groupId>org.zanata</groupId>
         <artifactId>zanata-maven-plugin</artifactId>
-        <version>3.8.0</version>
+        <version>${project.version}</version>
```

Comment:
I think `project.version` actually == 0 in these demo projects. They aren't part of the normal reactor build.